circular_linked_list/circular_linked_list.cpp

```diff
@@ -0,0 +1,45 @@
+#include<iostream>
+
+using namespace std;
+
+struct node{
+    int data;
+    node* next;
+};
```

Comment:
Line contains following spacing inconsistencies:
- Tabs used instead of spaces.

*Origin: SpaceConsistencyBear, Section: `all.space_checks`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpdf9_8a8n/circular_linked_list/circular_linked_list.cpp
+++ b/tmp/tmpdf9_8a8n/circular_linked_list/circular_linked_list.cpp
@@ -11,7 +11,7 @@
     node *current = head;
     cout << "The list is ";
     do {
-   		cout << current->data << " ";
+        cout << current->data << " ";
         current = current->next;
     } while (current != head);
     cout << endl;
```